packages/editor/src/components/post-preview-button/index.js

```diff
@@ -90,8 +90,8 @@ export class PostPreviewButton extends Component {
 
 		const markup = `
 			<div class="editor-post-preview-button__interstitial-message">
-				<p>Please wait&hellip;</p>
-				<p>Generating preview.</p>
+				<p>${ escape( __( 'Please wait&hellip;' ) ) }</p>
```

Comment:
The result of ` _.escape( 'Please wait&hellip;' );` is `Please wait&amp;hellip;`, which will effectively print to the screen as `Please wait&hellip;`.